src/main/java/org/junit/contrib/java/lang/system/internal/LogPrintStream.java

```diff
@@ -67,23 +67,27 @@ public String getLogWithNormalizedLineSeparator() {
 	}
 
 	public void mute() {
-		muteableLogStream.originalStreamMuted = true;
+		mutableLogStream.originalStreamMuted = true;
+	}
+	public void unTrackOriginalStream(){
+		mutableLogStream.trackOriginalStream = false;
 	}
 
 	public void muteForSuccessfulTests() {
 		mute();
-		muteableLogStream.failureLogMuted = false;
+		mutableLogStream.failureLogMuted = false;
 	}
 
-	private static class MuteableLogStream extends OutputStream {
+	private static class MutableLogStream extends OutputStream {
```

Comment:
Good catch! Can you please create a stand-alone pull request that renames `MuteableLogStream` to `MutableLogStream`. I like to assign changes that belong together to a single commit.